### PR TITLE
Update mailmap and zenodo for 4.1

### PR DESCRIPTION
**Pull request by dstansby**
I'll self merge this (if there are no objections to that?) when I do feature freeze to make sure everyone is included.

### CODE REVIEW

**Comment by dstansby** [comment]
👋 hello new contributors! I am updating the list of authors for the zenodo entry (see https://zenodo.org/record/3779284 for a previous example). I have mentioned you because you are a new contributor since sunpy 4.0.

If you would like, please leave a comment below with:

The preferred name you would like to be listed as
Your ORCID
Your affiliation

I will then manually add this to the information. Note that none of this is compulsory!

@mwhv2 @mariusgiger @naveensrinivasan @WilliamJamieson @tal66

**Comment by WilliamJamieson** [comment]
Preferred Name: William Jamieson
ORCID: [0000-0001-5976-4492](https://orcid.org/0000-0001-5976-4492)
Affiliation: Space Telescope Science Institute

**Comment by mariusgiger** [comment]
Thanks @dstansby!

Preferred Name: Marius Giger
ORCID: [0000-0002-6863-6502](https://orcid.org/0000-0002-6863-6502)
Affiliation: University of Applied Sciences and Arts Northwestern Switzerland

**Comment by mwhv2** [comment]
Preferred Name: Matt Wentzel-Long
ORCID: [0000-0002-3106-4598](https://orcid.org/0000-0002-3106-4598)
Affiliation: Stark State College

Thank you @dstansby!

**Comment by tlml** [comment]
Oops, thanks for the heads-up!

The preferred name: Timo Laitinen
Your ORCID: 0000-0002-7719-7783
Your affiliation: Jeremiah Horrocks Institute, University of Central Lancashire, UK

File: tools/update_zenodo.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def remove_initials(name):
 # Remove initials
 authors = [remove_initials(auth) for auth in authors]
 # List of authors to ignore because they are bots
-manual_ignore = ['codetriage-readme-bot', 'sunpy-backport[bot]', 'pre-commit-ci[bot]']
+manual_ignore = ['codetriage-readme-bot', 'sunpy-backport[bot]', 'pre-commit-ci[bot]', 'dependabot[bot]']
```
Comment (nabobalis) [inline tools/update_zenodo.py:26]: BOTS COUNT

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Merge when you need to.

**Pull request merged**